pages/project/read/content/index.js

```diff
@@ -0,0 +1,6 @@
+const { merge } = require('lodash');
+const baseContent = require('../../content');
+
+module.exports = merge({}, baseContent, {
+
+});
```

Comment:
Y THO?

Reply:
😆

Reply:
it will have content, but this page only exists in my brain, which doesn't contain any content